docs/src/pages/customization/color/color.md

```diff
@@ -92,9 +92,9 @@ These color palettes, originally created by Material Design in 2014, are compris
 Given a _HUE_ (red, pink, etc.) and a _SHADE_ (500, 600, etc.) you can import the color like this:
 
 ```jsx
-import HUE from '@material-ui/core/colors/HUE';
+import { red } from '@material-ui/core/colors';
 
-const color = HUE[SHADE];
+const color = red[500];
 ```
```

Comment:
Interesting change, it's not clear to me which form or clearer, why not.

Reply:
Generic + specific example is usually the safest way.